### ISSUE TITLE
Swift problem!

### ISSUE THREAD

**Issue by dinhcuong1989** [leverdeterre/JMActionSheetDescription#5]
Im trying to use JMACtionSheet in swift but i get an error when perform action. 
Here is my code: 

```
    let collectionItem: JMActionSheetCollectionItem = JMActionSheetCollectionItem()
    var collectionItems: [AnyObject] = [AnyObject]()
    var item: JMCollectionItem = JMCollectionItem()
    item.actionName = "facebook"
    item.actionImage = UIImage(named: "fb-icon3-150x150.png")
    collectionItems.append(item)
    item = JMCollectionItem()
    item.actionName = "AppStore"
    item.actionImage = UIImage(named: "apple_store_app_icon.jpg")
    collectionItems.append(item)
    item = JMCollectionItem()
    item.actionName = "Password"
    item.actionImage = UIImage(named: "1Pi-icon-1024-150x150.png")
    collectionItems.append(item)
    item = JMCollectionItem()
    item.actionName = "Flipboard"
    item.actionImage = UIImage(named: "Flipboard-App-Icon-150x150.jpg")
    collectionItems.append(item)
    item = JMCollectionItem()
    item.actionName = "Gmail"
    item.actionImage = UIImage(named: "gmail-ios-icon-app-150x150.png")
    collectionItems.append(item)
    item = JMCollectionItem()
    item.actionName = "Mail"
    item.actionImage = UIImage(named: "mail-iphone-150x150.jpg")
    collectionItems.append(item)
    collectionItem.elements = collectionItems
    collectionItem.collectionActionBlock =  {(selectedValue: JMCollectionItem) -> Void in
        print("selectedValue %@", selectedValue.actionName)
    }

    let desc: JMActionSheetDescription = JMActionSheetDescription()
    desc.actionSheetTintColor = UIColor.grayColor()
    desc.actionSheetCancelButtonFont = UIFont.boldSystemFontOfSize(17.0)
    desc.actionSheetOtherButtonFont = UIFont.systemFontOfSize(16.0)
    desc.title = "Select a value"
    let cancelItem: JMActionSheetItem = JMActionSheetItem()
    cancelItem.title = "Cancel"
    desc.cancelItem = cancelItem

    desc.items = [collectionItem]
    JMActionSheet.showActionSheetDescription(desc, inViewController: self, fromView: sender, permittedArrowDirections: .Any)
```

and i get error on line 
collectionItem.collectionActionBlock =  {(selectedValue: JMCollectionItem) -> Void in
            print("selectedValue %@", selectedValue.actionName)
        }
 error: 
Cannot assign value of type '(JMCollectionItem) -> Void' to type 'JMActionSheetSelectedItemBlock!'

If i delete the line, all work fine. but can not perform any action

**Comment by leverdeterre** [comment]
Hi, 
i have reproduce and fix this issue in the 0.4.7

My demo code 
class YourActionSheetCollectionItem : NSObject, JMActionSheetCollectionItemProtocol {
    var actionName: String!
    var actionImage: UIImage!
}

```
   let collectionItem: JMActionSheetCollectionItem = JMActionSheetCollectionItem()

    var collectionItems: [AnyObject] = [AnyObject]()
    var item: YourActionSheetCollectionItem = YourActionSheetCollectionItem()
    item.actionName = "facebook"
    item.actionImage = UIImage(named: "apple_store_app_icon.jpg")
    collectionItems.append(item)

    item = YourActionSheetCollectionItem()
    item.actionName = "AppStore"
    item.actionImage = UIImage(named: "apple_store_app_icon.jpg")
    collectionItems.append(item)

    item = YourActionSheetCollectionItem()
    item.actionName = "Password"
    item.actionImage = UIImage(named: "apple_store_app_icon.jpg")
    collectionItems.append(item)

    item = YourActionSheetCollectionItem()
    item.actionName = "Flipboard"
    item.actionImage = UIImage(named: "apple_store_app_icon.jpg")
    collectionItems.append(item)

    item = YourActionSheetCollectionItem()
    item.actionName = "Gmail"
    item.actionImage = UIImage(named: "apple_store_app_icon.jpg")
    collectionItems.append(item)

    item = YourActionSheetCollectionItem()
    item.actionName = "Mail"
    item.actionImage = UIImage(named: "apple_store_app_icon.jpg")
    collectionItems.append(item)
    collectionItem.elements = collectionItems

    let block = {(item :AnyObject!) -> Void in
        print("selectedValue /(item/))")
    }
    collectionItem.collectionActionBlock = block

    let desc: JMActionSheetDescription = JMActionSheetDescription()
    desc.actionSheetTintColor = UIColor.grayColor()
    desc.actionSheetCancelButtonFont = UIFont.boldSystemFontOfSize(17.0)
    desc.actionSheetOtherButtonFont = UIFont.systemFontOfSize(16.0)
    desc.title = "Select a value"
    let cancelItem: JMActionSheetItem = JMActionSheetItem()
    cancelItem.title = "Cancel"
    desc.cancelItem = cancelItem
    desc.items = [collectionItem]
    JMActionSheet.showActionSheetDescription(desc, inViewController: self.navigationController, fromView: self.navigationController!.view, permittedArrowDirections: .Any)
```

**Comment by dinhcuong1989** [comment]
Now i got an error when i update to Xcode 8

> let block = {(item :AnyObject!) -> Void in
>             print("selectedValue /(item/))")
>         }
> collectionItem.collectionActionBlock = block as! JMActionSheetSelectedCollectionItemBlock

Build fine but crash when i click on item or action.

If i remove "as! JMActionSheetSelectedCollectionItemBlock" i got an error and i cant build.
